jetstream/jetstream.go

```diff
@@ -199,6 +199,14 @@ type (
 	}
 )
 
+const (
```

Comment:
Those constants (other than `_EMPTY_` are not used) in `jetstream` package, so should not be added preemptively. Additionally, it was a conscious decision not to use `_EMPTY_` in this new package - it is not idiomatic in go to use constants for zero values + it does not follow const naming guidelines.

I would suggest dropping those constants and simply doing `if str == "" {}` in code.